pkg/controller/controller_instance_test.go

```diff
@@ -701,6 +743,11 @@ func TestReconcileInstanceNamespaceError(t *testing.T) {
 	// One single action comes from getting namespace uid
 	kubeActions := fakeKubeClient.Actions()
 	assertNumberOfActions(t, kubeActions, 1)
```

Comment:
I think you can remove assertNumberOfActions since `checkKubeClientActions` makes the same assertion.